app/queue/queue.py

```diff
@@ -150,14 +150,16 @@ def handle_message(message, event_producer):
     with PayloadTrackerContext(
         payload_tracker, received_status_message="message received", current_operation="handle_message"
     ):
-        (output_host, add_results) = add_host(validated_operation_msg["data"])
+        (output_host, insights_id, add_results) = add_host(validated_operation_msg["data"])
```

Comment:
Here, the added Host ID is returned.

```suggestion
        output_host, host_id, insights_id, add_results = add_host(validated_operation_msg["data"])
```